pkg/graph/schema.graphql

```diff
@@ -713,8 +713,8 @@ input SystemIntakeGRBPresentationLinksInput {
   recordingLink: String
```

Comment:
I think we discussed making all fields omittable instead of just optional. Currently, you can't truly clear the value, just set it to empty string. Ideally, we should be able to tell if a value is present and null, vs absent.